cryptocurrency/README.md

```diff
@@ -0,0 +1,17 @@
+# cryptocurrency
+
+The script has will pull live and historical for specifed cryptocurencies from the [CoinGecko](https://www.coingecko.com/) website into the Deephave Community Edition IDE.
+
+## Variables ##
+- **timeToWatch:** Integer number of minutes to run the script
+- **secondsToSleep:** Integer number of seconds between data pulls.  Note that too frequent of requests will generate a HTTP status '429 Too Many Requests'. Recommend this number of 10 or higher.
+- **getHistory:** Boolean value, if one wants only live data or live and historical
+- **daysHistory:** Integer number of days to collect history. Data has automatic granularity. Minutely data will be used for duration within 1 day, Hourly data will be used for duration between 1 day and 90 days, Daily data will be used for duration above 90 days.
```

Comment:
```suggestion
- **daysHistory:** Integer number of days to collect history. Data has automatic granularity. 
   - Minutely data will be used for a duration within 1 day.
   - Hourly data will be used for a duration between 1 day and 90 days.
    - Daily data will be used for a duration above 90 days.
```

Reply:
Is minutely a typical word?

Reply:
it is a word. Even looked it up :) https://english.stackexchange.com/questions/3091/weekly-daily-hourly-minutely